types/sbd/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for sbd 1.0
+// Project: http://github.com/tessmore/sbd
+// Definitions by: Brian Cort <https://github.com/thatcort>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace sbd;
```

Comment:
This line seems can be removed

Reply:
When I remove it, WebStorm no longer autocompletes as expected:

`import * as tokenizer from 'sbd';
tokenizer.sen...
`
With the namespace declaration, WebStorm autocompletes `sentences`. Without, it doesn't.

This may just be a bug in WebStorm, but it improved my developer experience.

Happy to remove the line if you prefer.